embedchain/utils.py

```diff
@@ -149,6 +148,10 @@ def is_openapi_yaml(yaml_content):
             logging.debug(f"Source of `{formatted_source}` detected as `notion`.")
             return DataType.NOTION
 
+        if url.netloc in {"substack.com"}:
+            logging.debug(f"Source of `{formatted_source}` detected as `substack`.")
+            return DataType.SUBSTACK
```

Comment:
What if someone has a custom domain name for substack url? Do they have to pass the `data_type=substack`? 

Also, if I am trying to just load a particular substack post as a webpage, that will be loaded as substack data type, right?

Reply:
Yeah! you're correct, should these be removed from auto-detecting types?